openapi/components/schemas/GroupPost.yaml

```diff
@@ -0,0 +1,41 @@
+title: GroupPost
+type: object
+properties:
+  id:
+    $ref: ./NotificationID.yaml
+    nullable: true
+  groupId:
+    $ref: ./GroupID.yaml
+    nullable: true
+  authorId:
+    $ref: ./UserID.yaml
+    nullable: true
+  editorId:
+    $ref: ./UserID.yaml
+    nullable: true
+  visibility:
+    $ref: ./GroupPostVisibility.yaml
+    nullable: false
```

Comment:
Nullable doesn't need to be specified for non-nullable fields.

Reply:
The line needs to be deleted?

Reply:
Yes & the other lines like it.

Reply:
Changed with https://github.com/vrchatapi/specification/pull/315/commits/5b368171b2eacfbc879dabcb51f9a0729ddcd39e , only 3 nullable fields available. I see the editorId field but I don't see a front end method to using it. Might be a WIP on VRChat side. I'll work on that once it's out because it's of interest to my tool.